Lib/ufo2ft/filters/dottedCircleFilter.py

```diff
@@ -0,0 +1,158 @@
+import logging
+import math
+
+from ufoLib2.objects import Glyph
+
+from ufo2ft.featureCompiler import parseLayoutFeatures
+from ufo2ft.featureWriters import ast
+from ufo2ft.filters import BaseFilter
+from ufo2ft.util import _GlyphSet, _LazyFontName
+
+logger = logging.getLogger(__name__)
+CIRCULAR_SUPERNESS = 0.551784777779014
```

Comment:
what does 'circular superness' means? a comment might help

Reply:
`Did you mean 'Circular Super NES'?` ;-)
![Circular_SuperNES](https://user-images.githubusercontent.com/213676/163882054-3952d958-371d-41d0-934d-0ab048363b3b.png)
(CC-BY Jonny Cantão: https://www.iconspng.com/image/74309/mini-snes)

Reply:
Heh. I added a link to https://pomax.github.io/bezierinfo/#circles_cubic